chart/values.yaml

```diff
@@ -4,7 +4,7 @@ global:
   arch: "amd64"
   type: "k8s"
   namespaceYaml: true
-
+  affinity: {}
```

Comment:
Please provide an example in a comment block on how to set the affinity values

Reply:
is this acceptable as an example ?